google/auth/credentials.py

```diff
@@ -11,19 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
```

Comment:
Avoid committing unrelated whitespace changes, it makes the diff larger.

Reply:
Hi Carl, those are auto generated from lint fix tools since we failed the previous lint check.